packages/clarity-js/types/layout.d.ts

```diff
@@ -58,7 +58,7 @@ export interface NodeInfo {
 export interface NodeValue {
     id: number;
     parent: number;
-    next: number;
+    previous: number;
```

Comment:
Definitely not backwards compatible. Needs a major version bump (0.5.0)

Reply:
It is backward compatible. This is an internal detail to decode & js that by design are supposed to be tightly coupled. No impact to any other layer consuming this data. Happy to discuss offline if we should make a major version bump or not.